rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
```

Comment:
In logrus, we found that having the logger be exportable as an `io.Writer`, and thus pluggable in the stdlib `log.Logger` was a good way to remain compatible with things that expect a stdlib logger.  But otherwise, the stdlib logger is very limited for structured logging.